packages/ckb/src/utils/cell-dep.ts

```diff
@@ -31,14 +41,37 @@ const GITHUB_CELL_DEPS_JSON_URL =
 const CDN_GITHUB_CELL_DEPS_JSON_URL =
   'https://cdn.jsdelivr.net/gh/utxostack/typeid-contract-cell-deps@main/deployment/cell-deps.json';
 
-const request = (url: string) => axios.get(url, { timeout: 5000 });
+const VERCEL_CELL_DEPS_JSON_STATIC_URL = 'https://typeid-contract-cell-deps.vercel.app/deployment/cell-deps.json';
+
+const VERCEL_SERVER_CELL_DEPS_JSON_URL = 'https://typeid-contract-cell-deps.vercel.app/api/cell-deps';
 
-export const fetchCellDepsJson = async () => {
+const request = (url: string) => axios.get(url, { timeout: 10000 });
+
+const fetchCellDepsJsonFromStaticSource = async () => {
   try {
-    const response = await Promise.any([request(CDN_GITHUB_CELL_DEPS_JSON_URL), request(GITHUB_CELL_DEPS_JSON_URL)]);
+    const response = await Promise.any([
+      request(CDN_GITHUB_CELL_DEPS_JSON_URL),
+      request(GITHUB_CELL_DEPS_JSON_URL),
+      request(VERCEL_CELL_DEPS_JSON_STATIC_URL),
+    ]);
     return response.data as CellDepsObject;
   } catch (error) {
-    // console.error('Error fetching cell deps:', error);
+    // for (const e of error.errors) {
+    //   console.error('Error fetching cell deps from static source:', e);
+    // }
+  }
+};
+
+const fetchCellDepsJson = async () => {
+  try {
+    const response = await request(VERCEL_SERVER_CELL_DEPS_JSON_URL);
```

Comment:
```
{
  "compatibleXudt": {
    "0x1142755a044bf2ee358cba9f2da187ce928c91cd4dc8692ded0337efa677d21a": {
      "outPoint": {
        "index": "0x0",
        "txHash": "0xed7d65b9ad3d99657e37c4285d585fea8a5fcaf58165d54dacf90243f911548b"
      },
      "depType": "code"
    },
    "0x26a33e0815888a4a0614a0b7d09fa951e0993ff21e55905510104a0b1312032b": {
      "outPoint": {
        "index": "0x0",
        "txHash": "0x8ec1081bd03e5417bb4467e96f4cec841acdd35924538a35e7547fe320118977"
      },
      "depType": "code"
    }
  }
}
```

The return object of `https://typeid-contract-cell-deps.vercel.app/api/cell-deps` is NOT consistent with `https://typeid-contract-cell-deps.vercel.app/deployment/cell-deps.json`

Reply:
Fixed with `typeid-contract-cell-deps` https://github.com/utxostack/typeid-contract-cell-deps/pull/17